index.html

```diff
@@ -482,6 +482,27 @@ <h3>
           Details of how to actually get the [=digital credential=] are
           forthcoming.
         </aside>
+        <ol class="algorithm" >
+          <li>
+            <aside class="issue">
+              The following guidance on handling multiple requests will need to be relocated once
+              this entire section of the algorithm is defined.
+            </aside>
+            When multiple [=digital credential/presentation requests=] are present
+            in {{DigitalCredentialRequestOptions/requests}},
+            [=user agents=] MUST only return one
+            [=digital credential/presentation responses | presentation response=] for one
+            of these requests.
```

Comment:
```suggestion
            [=user agents=] MUST return a maximum of one
            [=digital credential/presentation responses | presentation response=] for each
            of these requests.
```

Reply:
This is slightly different behavior than what I'm trying to define here. I'm trying to capture the idea that, while a verifier may specify multiple requests using different protocols for each, the response the verifier gets back will only be for one of those requests.

The suggested wording sounds like it'd open the door to allowing multiple requests to get back one response each. Am I misunderstanding the intent of these proposed changes?

Reply:
Perhaps this?
```suggestion
            [=user agents=] MUST NOT return more than one
            [=digital credential/presentation responses | presentation response=], regardless
            of the number of requests received.
```

Reply:
@TallTed I tweaked the text a bit into something I think is a midpoint between our positions. What do you think?

Reply:
Oh apologies, this review doesn't appear in the Files changed tab anymore because I made enough changes to everything. I'll resolve this and tag you in a new comment.